src/App.tsx

```diff
@@ -1,7 +1,42 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  hasPressedKey: boolean,
+  pressedKey: string,
+};
+
+export class App extends React.Component<{}, State> {
+  state = {
+    hasPressedKey: false,
+    pressedKey: '',
```

Comment:
Do you really need both of them? 
If `pressedKey` is not an empty string, then apparently some key was pressed 🤔